src/style.css

```
@@ -1 +1,80 @@
 /* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.navbar {
+  display: flex;
+  padding: 0 50px;
+  height: 60px;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  align-self: center;
+  height: 40px;
+  width: 40px;
+}
+
+li {
+  list-style-type: none;
+}
+
+.menu {
+  display: flex;
+  padding: 0;
+  margin-top: 19px;
+}
+
+.item {
+  align-items: center;
+  text-decoration: none;
+  margin-right: 20px;
+  line-height: 14px;
+  color: black;
+  padding: 23px 0;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.video {
+  margin-right: 0;
+}
```

```suggestion
.video {
  margin-right: 0;
}
```
замість створення окремого класу, використовую для скидання марджину псевдоклас :last-child